src/OpenConext/AttributeAggregationApiClientBundle/Repository/AttributeAggregationRepository.php

```diff
@@ -45,8 +46,11 @@ public function findAllFor(
     public function unsubscribeAccount(
         int $accountId,
     ): bool {
-        $result = $this->apiClient->delete('disconnect/%d', [$accountId]);
-
-        return isset($result->status) && $result->status === 'OK';
+        try {
+            $this->apiClient->delete('disconnect/%d', [$accountId]);
+            return true;
```

Comment:
What if the API retusn a result, but the status !== 'OK'. For example it was unable to remove the accoutn for some good reason.

Profile would now interpret that as if the delete was a success.